pkg/commands/rules.go

```diff
@@ -152,7 +152,7 @@ func (r *RuleCommand) Register(app *kingpin.Application) {
 	).StringVar(&r.RuleFilesPath)
 
 	// Prepare Command
-	prepareCmd.Arg("rule-files", "The rule files to check.").Required().ExistingFilesVar(&r.RuleFilesList)
+	prepareCmd.Arg("rule-files", "The rule files to check.").ExistingFilesVar(&r.RuleFilesList)
```

Comment:
what does this change do?

Reply:
The argument to the command (e.g. `cortextool rules prepare file1.yml` should be options. Given you can use the flags to pass a directory to scan (e.g. `cortextool rules prepare --rules-dir=output/`).